### PR TITLE
refactor(pageination): added only page nr to pageination repsonse

### PR DESCRIPTION
**Pull request by Her0elt**
# refactored basepageination
## added only page nr to pageination repsonse you still access the next page my using the query param page=page_nr but frontend now only get the page nr so they dont have to use a hacky method to get the page nr
Link to Trello task: ...

Comments/issues:

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] PR name is descriptive
- [x] The PR includes the link to the relevant Trello task
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)

- [x] `python manage.py runserver` was run locally witout errors

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

File: app/content/pagination.py [hunk 1]
```diff
@@ -1,6 +1,31 @@
...
+                ]
+            )
+        )
+
+    def get_next_page(self):
+        if not self.page.has_next():
+            return None
+        return self.page.next_page_number()
+
+    def get_previou_page(self):
```
Comment (Eirsteir) [inline app/content/pagination.py:28]: Typo in method name 😊

File: app/content/pagination.py [hunk 2]
```diff
@@ -1,6 +1,31 @@
...
+        if not self.page.has_previous() or self.page.previous_page_number() < 1:
```
Comment (Eirsteir) [inline app/content/pagination.py:29]: Isn't the second statement in this if-statement essentially the same as the first?

**Review by Eirsteir — CHANGES_REQUESTED** [review]
Good work Hermann, only some minor comments this time 😉

File: app/content/pagination.py [hunk 3]
```diff
@@ -1,6 +1,35 @@
 from rest_framework.pagination import PageNumberPagination
+from rest_framework.response import Response
+from collections import OrderedDict
+
+from rest_framework.utils.urls import remove_query_param, replace_query_param
 
 
 class BasePagination(PageNumberPagination):
-    page_size = 25
+    page_size = 1
```
Comment (Eirsteir) [inline app/content/pagination.py:8]: This should still be 25, shouldn't it?

**Review by Eirsteir — APPROVED** [review]
Looks good to me!

**Comment by Eirsteir** [comment]
What is the status on this @Her0elt?

**Comment by Her0elt** [comment]
@Eirsteir wating on frontend to change pageination der

**Pull request merged**